src/core_ocean/Registry.xml

```diff
@@ -239,7 +239,19 @@
 	<nml_record name="ALE_vertical_grid" mode="forward">
 		<nml_option name="config_vert_coord_movement" type="character" default_value="uniform_stretching" units="unitless"
 					description="Determines the vertical coordinate movement type. 'uniform_stretching' distributes SSH perturbations through all vertical levels (z-star vertical coordinate); 'fixed' places them all in the top level (z-level vertical coordinate); 'user_specified' allows the input file to determine the distribution using the variable vertCoordMovementWeights (weighted z-star vertical coordinate); and 'impermeable_interfaces' makes the vertical transport between layers zero, i.e. $w^t=0$ (idealized isopycnal)."
-					possible_values="'uniform_stretching', 'fixed', 'user_specified', 'impermeable_interfaces'"
+					possible_values="'uniform_stretching', 'fixed', 'user_specified', 'impermeable_interfaces', 'tapered'"
+		/>
+		<nml_option name="config_ALE_thickness_proportionality" type="character" default_value="restingThickness_times_weights" units="unitless"
+					description="When config_vert_coord_movement='uniform_stretching' (z-star type coordinate), determines whether ALE layer thickness is proportional to the resting thickness times weights, or just the weights. The first is standard for global runs and is what is specified in Petersen et al 2015 eqns 4 and 6. The second is useful for wetting/drying test cases where resting thickness may be zero at the coastlines."
+					possible_values="'restingThickness_times_weights' or 'weights_only'"
```

Comment:
@pwolfram I realized that we don't want this alteration in general, so I am protecting it with this flag. In general in the ocean, we need to have the thickness vary in proportion to the resting thickness (i.e. original distribution), not the weights. You can run wetting/drying test cases with the 
`config_ALE_thickness_proportionality = weights_only` 
setting. But this breaks ice shelf cavities, as @xylar feared, because you need to 'squeeze the accordion' in proportion to the original thicknesses, especially for extreme compression, like we have under ice shelves.

This commit is now bit-for-bit with previous for MPAS-O and E3SM. We will need to think about this more carefully to put wetting/drying along coastlines and severe SSH alterations below ice shelves in the same domain.